presto-main/src/main/java/com/facebook/presto/sql/planner/plan/DistinctLimitNode.java

```diff
@@ -43,13 +43,15 @@ public DistinctLimitNode(
             @JsonProperty("source") PlanNode source,
             @JsonProperty("limit") long limit,
             @JsonProperty("partial") boolean partial,
+            @JsonProperty("distinctSymbols") List<Symbol> distinctSymbols,
```

Comment:
there should be a convention (and check state) probably that `distinctSymbols` either include or exclude `hashSymbol`